Assets/Scripts/Core/Import.cs

```diff
@@ -7,18 +7,25 @@ public static List<Star> ImportStarData(string sourceData)
         List<Star> stars = new List<Star>();
         foreach (var line in splitToLines(sourceData))
         {
-            if (!line.StartsWith("Constellation"))
+            if (!line.StartsWith("Hip"))
             {
                 string[] values = line.Split('\t');
-                if (values.Length == 6)
+
+                if (values.Length == 11)
                 {
-                    string constellation = values[0];
-                    string XByerFlamsteed = values[1];
-                    float RA = float.Parse(values[2]);
-                    float Dec = float.Parse(values[3]);
-                    float Dist = float.Parse(values[4]);
-                    float Mag = float.Parse(values[5]);
-                    Star star = new Star(constellation, XByerFlamsteed, RA, Dec, Dist, Mag);
+                    int Hip = int.Parse(values[0]);
+                    string constellation = values[1];
```

Comment:
Would be nice to change capitalization to match all other properties that start with a capital letter

Reply:
Agree.  Fixed.